scripts/_common.sh

```diff
@@ -4,11 +4,11 @@
 # COMMON VARIABLES
 #=================================================
 
-#REMOVEME? YNH_PHP_VERSION="8.1"
+phpversion="8.2"
```

Comment:
```suggestion
```

(There should be no need to define this variable, it's already a setting defined in the core during the apt resource)